README.md

```diff
@@ -0,0 +1,35 @@
+# PagerDuty Short Circuiter
+This project contains PagerDuty CLI to reduce the time taken from when SRE receives a PD alert to when troubleshooting on the cluster actually begins.
+
+- The document of the project design can be found [here](https://docs.google.com/document/d/1VV3bN3WBI-DrJ59jOciA5tyvnxhldSVW-lDGNu2ONiw/edit?usp=sharing).
+- Jira Epics for this project can be found [here](https://issues.redhat.com/browse/OSD-8102).
```

Comment:
Oops my bad, since this is open source, mentioning Jira is not a good idea. 

Could you remove this link